### DISCUSSION TITLE
Why clicking suggestion in spell check no longer works?

### DISCUSSION THREAD

**Discussion by devkatalist** [SubtitleEdit/subtitleedit#8201] [Q&A]
When I used to click a suggestion, it acted as "Use". Now it does nothing. Why?

![Capture d'écran 2024-04-16 192717](https://github.com/SubtitleEdit/subtitleedit/assets/37704802/b52a87c6-b70d-41ba-bcbe-f259e697f4c3)

**Comment by niksedk** [comment]
What version of SE are you using?
Hm, seems to work here. Perhaps it's something with the subtitle file. Could you attach it (GitHub supports .zip files) ?

**Reply by devkatalist** [reply]
[3808050_fr_FR_04_06_VTmp4.fr.srt.zip](https://github.com/SubtitleEdit/subtitleedit/files/15001488/3808050_fr_FR_04_06_VTmp4.fr.srt.zip)
